src/feedback.css.ts

```diff
@@ -33,41 +37,65 @@ export const innerContainer = style({
     padding: '64px 0px 16px',
     zIndex: 1, // needed to support hack for o2-classic
     position: 'relative', // needed to support hack for o2-classic
+    display: 'flex',
+    flexDirection: 'column',
+    '@media': {
+        [mq.desktopOrBigger]: {
+            paddingBottom: 40,
+        },
+    },
 });
 
 export const iconContainer = sprinkles({
     width: 48,
     height: 48,
 });
 
-export const feedbackDataAppear = style({
-    opacity: 0,
-    transform: 'translateY(24px)',
-    '@media': {
-        [mq.desktopOrBigger]: {
-            transform: 'translateY(40px)',
-        },
+const textAppearAnimation = keyframes({
+    from: {
+        opacity: 0,
+        transform: 'translateY(24px)',
+    },
+    to: {
+        opacity: 1,
+        transform: 'translateY(0)',
     },
 });
 
-const feedbackDataAppearActive = style({
-    transitionProperty: 'opacity, transform',
-    transitionDuration: '1s',
-    transitionTimingFunction: 'cubic-bezier(0.215, 0.61, 0.355, 1)',
-    opacity: 1,
-    transform: 'none',
+const textAppearAnimationDesktop = keyframes({
+    from: {
+        opacity: 0,
+        transform: 'translateY(40px)',
```

Comment:
Maybe we could use variables for these hardcoded numbers. If in the future we need to change this, we would need to review each line of this file carefully otherwise.

Reply:
this 40 isn't related with any other number/style in the file